test_module.py

```
@@ -0,0 +1,21 @@
+"""A test module with various utilities."""
+
+
+def utility_function():
+    """A simple utility function."""
+    return "utility result"
+
+
+class TestClass:
+    """A test class."""
+
+    def __init__(self, value):
```

Ruff (0.11.9)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)